Automattic/woocommerce-payments
# Add a disputes task to the "Things to do next" list on WC > Home

Fixes #4341

#### Changes proposed in this Pull Request

This PR adds a task item to the **WC → Home** matching the one on **Payments → Overview**. 

<img width="740" alt="Screen Shot 2022-07-07 at 2 19 52 pm" src="https://user-images.githubusercontent.com/8490476/177690418-7f97ceed-7987-475b-878e-72af30060243.png">

#### Testing instructions

1. Purchase a product using a disputed payment method. eg `4000000000000259`
2. Go to **WooCommerce → Home** on your admin dashboard. 
3. Note the task item referencing the disputed payment. 
4. Clicking this task should navigate to the Disputes list table with the **"Needs response"** filter applied. 

-------------------


**Post merge**
